src/shared/dissect-image.c

```diff
@@ -2057,7 +2057,7 @@ static int verity_partition(
                         if (r == 0) {
                                 /* devmapper might say that the device exists, but the devlink might not yet have been
                                  * created. Check and wait for the udev event in that case. */
-                                r = device_wait_for_devlink(node, "block", 100 * USEC_PER_MSEC, NULL);
+                                r = device_wait_for_devlink(node, "block", 30 * USEC_PER_SEC, NULL);
```

Comment:
Uh this is way way too long for my use case, fast fallback is very important and 100ms was already quite high - note that this happens all the time because the kernel device-mapper is just bad

Reply:
So, WDYT about using a environment variable?

Reply:
that would work, but can I set it for pid1?

Reply:
or what about, leave default as is, and set the env var in the test?

Reply:
Yes, that's mostly for tests.